feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/weld/jpa/main/module.xml

```diff
@@ -45,8 +45,9 @@
         <module name="org.jboss.msc"/>
         <module name="javax.enterprise.api"/>
         <module name="org.wildfly.cdi-api-bridge"/>
+        <module name="org.wildfly.transaction.client"/>
         <module name="javax.persistence.api"/>
-        <module name="javax.transaction.api"/>
+        <module name="javax.transaction.api"/>        
```

Comment:
@scottmarlow This space probably should not be there?

Reply:
thanks, I missed that, removed the unneeded trailing space.